front-end/src/app/app-routing.module.ts

```diff
@@ -20,13 +23,16 @@ const routes: Routes = [
   { path: 'paciente-criar', component: CreatePacienteComponent },
   { path: 'hospital-criar', component: CreateHospitalComponent },
   { path: 'especialidade-criar', component: CreateEspecialidadeComponent },
-  { path: 'especialidade-editar', component: EditEspecialidadeComponent },
+  { path: 'especialidade-editar/:id', component: EditEspecialidadeComponent },
   { path: 'especialidade-listar', component: ListEspecialidadeComponent },
   { path: 'login', component: LoginComponent },
   { path: 'medico-listar', component: ListMedicoComponent },
   { path: 'medico-editar/:id', component: EditMedicoComponent },
   { path: 'paciente-editar/:id', component: PacienteEditarComponent },
   { path: 'paciente-listar', component: PacienteListarComponent },
+  { path: 'adm', component: AdmComponent},
```

Comment:
mano poe um nome melhor, pelo menos por o nome completo, no componente tb

Reply:
vou remover este coponente. No outro branch to fazendo um drop down menu, "Painel Administrativo -->Relatórios-->Estatísticas"